src/main/resources/native/android/AndroidManifest.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version='1.0'?>
-<manifest xmlns:a='http://schemas.android.com/apk/res/android' package='com.gluonhq.helloandroid' a:versionCode='1' a:versionName='1.0'>
```

Comment:
I don't think renaming a to android adds anything, so can this be reverted?

Reply:
sure

Reply:
done